.docker/orbit.Dockerfile.2022.1.1-ubuntu20.04

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Reference:
+# https://github.com/NVIDIA-Omniverse/IsaacSim-dockerfiles/blob/main/Dockerfile.2022.1.1-ubuntu20.04
+#
+# Build the image:
+# docker-compose build
+#
+# Run the container:
+# docker-compose up
+#
+# More info:
+# https://isaac-orbit.github.io/orbit/source/setup/installation.html
+#
+FROM isaac-sim:2022.1.1-ubuntu20.04
+
+# Install dependencies
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    build-essential \
+    cmake \
+    git \
+    ncurses-term \
+    python3-pip \
```

Comment:
System-wide `pip` should no longer be necessary with https://github.com/NVIDIA-Omniverse/Orbit/pull/28 (not tested).

```suggestion
```